### PR TITLE
[MAINT] remove non-existent SkipTest usage in example file

### PR DESCRIPTION
**Pull request by jbrockmendel**
trim trailing whitespace

### CODE REVIEW

File: examples/python/generic_mle.py [hunk 1]
```diff
@@ -103,13 +103,13 @@ def _ll_nb2(y, X, beta, alph):
 class NBin(GenericLikelihoodModel):
     def __init__(self, endog, exog, **kwds):
         super(NBin, self).__init__(endog, exog, **kwds)
-
```
Comment (jbrockmendel) [inline examples/python/generic_mle.py]: @josef-pkt would fixing this be more palatable if also fixed in the corresponding ipynb file?

**Comment by jbrockmendel** [comment]
@josef-pkt SkipTest got removed a few weeks ago and one usage fell through the cracks.  This fixes that.

**Comment by josef-pkt** [comment]
fixing the skiptest is fine, but we don't touch the notebook exports to py files, until we have checked how to sync them

**Comment by jbrockmendel** [comment]
> but we don't touch the notebook exports to py files

Reverted.

**Comment by jbrockmendel** [comment]
@josef-pkt edited as requested, de facto green

**Comment by jbrockmendel** [comment]
@josef-pkt edited as requested and defacto green

**Comment by jbrockmendel** [comment]
Closing in favor of #5167.

**Pull request closed without merging**